cookbooks/adiabatic_boundary.prm

```diff
@@ -0,0 +1,80 @@
+set Dimension                              = 3
+
+set Use years in output instead of seconds = true
+set End time                               = 0
+set Nonlinear solver scheme                = single Advection, single Stokes
+set Nonlinear solver tolerance             = 1e-5
+set Linear solver tolerance                = 1e-5
+set Max nonlinear iterations               = 10 
+set Temperature solver tolerance           = 1e-4
```

Comment:
This is a much higher tolerance (8 orders of magnitude) than the default value (1e-12). Unless there is very specific reason to use this value, I suggest removing this line and running the model with the default value.